Content edits in the Angular 2 Guide.

cfranger opened the pull request:
@juleskremer Here are my edits to the "Forms" page.

juleskremer commented on public/docs/ts/latest/guide/forms.jade:
```diff
@@ -1,83 +1,90 @@
 
-  Any seasoned web developer can slap together an HTML form with all the right tags.
-  It's more challenging to create a cohesive data entry experience that guides the
+  Any seasoned web developer can create an HTML form with all the right tags.
```
Can we change the tone here to remove the "seasoned web developer" -> It is important to create a data entry experience that guides...

juleskremer commented on public/docs/ts/latest/guide/forms.jade:
```diff
@@ -1,83 +1,90 @@
 
-  *That* takes design skills that are, to be frank, well out of scope for this guide.
+  *That* takes design skills that are, to be frank, well out of scope for this page.
 
```
Can we change the tone here:  "Designing forms requires x skills which are out of scope for this page"

juleskremer commented on public/docs/ts/latest/guide/forms.jade:
```diff
@@ -1,83 +1,90 @@
+   scope for this page), as well as framework support for
+   *two-way data binding, change tracking, validation, and error handling*,
+   which you'll learn about on this page."
+   -->
```
yes please

juleskremer commented on public/docs/ts/latest/guide/forms.jade:
```diff
@@ -1,83 +1,90 @@
   :marked
-    That's not the only way to create a form but it's the way we'll cover in this guide.
+    That's not the only way to create a form but it's the way that's covered in this page.
+//    CF: Is this note necessary?
```
It is but it should be more descriptive that there two choices: template-driven and reactive.

cfranger commented:
@juleskremer thanks for your comments. I committed some updates based on your feedback. I have a few more questions in the page--please search for "CF:"

cfranger commented:
@juleskremer My latest commit is an edit of the Glossary page. I added several comments with questions. Search for "CF:". Thanks!

juleskremer commented on public/docs/ts/latest/glossary.jade:
```diff
@@ -150,10 +151,11 @@ a#aot
     the component class, thereby attaching to the class the essential component metadata
     that Angular needs to create a component instance and render it with its template
     as a view.
+    <!-- CF: What do "it" and "its" refer to? The component class, component metadata, or component instance? -->
```
the component with it's template.

juleskremer commented on public/docs/ts/latest/glossary.jade:
```diff
@@ -164,7 +166,10 @@ a#aot
     The practice of writing compound words or phrases such that each word is separated by a dash or hyphen (`-`).
-    This form is also known as [kebab-case](#kebab-case).
+    This form is also known as kebab-case.
+    <!-- CF: I removed the link to the kebab-case glossary entry because the entry has no additional
```
lgtm

juleskremer commented on public/docs/ts/latest/glossary.jade:
```diff
@@ -241,23 +250,26 @@ a#aot
     Angular provides and relies upon its own sophisticated
-    [dependency injection](!{docsLatest}/guide/dependency-injection.html) system
+    dependency-injection system
+    <!-- CF: I removed the link here because the same link is at the end of this definition. -->
```
lgtm

juleskremer commented on public/docs/ts/latest/glossary.jade:
```diff
@@ -618,22 +634,21 @@ a#snake-case
-
-    A service is a class with a focused purpose.
-    We often create a service to implement features that are
+    <!-- CF: Removed the first sentence/paragraph because it's redundant.
```
lgtm

juleskremer commented on public/docs/ts/latest/glossary.jade:
```diff
@@ -738,20 +754,22 @@ a#Y
+      of asynchronous activities, such as DOM events (for example, clicks),
       [promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise), and
       [XHR](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest)
+      <!-- CF: This link goes to a mozilla page titled "XMLHttpRequest". XHR is mentioned only twice, starting
```
yes

juleskremer commented on public/docs/ts/latest/guide/forms.jade:
```diff
@@ -437,64 +437,66 @@ figure.image-display
-  If we ignore the `pristine` state, we would hide the message only when the value is valid.
-  If we arrive in this component with a new (blank) hero or an invalid hero,
-  we'll see the error message immediately, before we've done anything.
+  <!-- CF: How about "developers" or "users" (not sure which is appropriate) instead of "folks"? -->
```
developers

juleskremer commented on public/docs/ts/latest/guide/forms.jade:
```diff
@@ -437,64 +437,66 @@ figure.image-display
+  you'll see the error message immediately, before you've done anything.
 
   Some folks find that behavior disconcerting.
+  <!-- CF: How about "developers" or "users" (not sure which is appropriate) instead of "folks"? -->
```
developers

cfranger commented:
@juleskremer thanks for your feedback. I've updated the "Forms" and "Glossary" pages with your feedback. I also fixed a few issues that I missed on my first pass.
